src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -0,0 +1,177 @@
+package reposense.parser;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import net.sourceforge.argparse4j.inf.ArgumentParserException;
+
+import reposense.frontend.CliArguments;
+
+public class ArgsParserTest {
+
+    private String projectDirectory;
+
+    public ArgsParserTest() {
+        projectDirectory = System.getProperty("user.dir");
+    }
+
+    private Date getDate(int year, int month, int date) {
+        return new Calendar.Builder()
+                            .setDate(year, month, date)
+                            .setTimeOfDay(0, 0, 0)
+                            .build()
+                            .getTime();
+    }
+
+    @Test
+    public void allCorrectInputs_success() throws ArgumentParserException {
+        Path configFilePath = Paths.get(projectDirectory + "/sample_small.csv");
```

Comment:
for `Paths`, I think we can use `Paths.get(projectDirectory, "sample_small.csv")`, instead of manually adding the file separator ourselves